versions.yml

```diff
@@ -4,6 +4,20 @@ logstash: 8.0.0
 logstash-core: 8.0.0
 logstash-core-plugin-api: 2.1.16
 
+bundled_jdk:
+  linux:
+      vendor: "adoptopenjdk"
+      revision: 14
```

Comment:
Are we wanting to bundle JDK 14? Or stick to LTS releases?

Reply:
we decided for LTS (11 at the moment)